src/components/AppFailed.tsx

```diff
@@ -1,8 +1,13 @@
+import localeTexts from "../common/locales/en-us";
+
 /**
  * AppFailed component.
  * This components will be rendered if the App fails to initialize.( ContentStack SDK)
  */
 export const AppFailed = () => {
+  const handleLearnMoreClick = () => {
+    window.open("https://www.contentstack.com/docs/developers/developer-hub/marketplace-app-boilerplate/", "_blank");
```

Comment:
take the URL from a variable like
appBoilerplateURL = https://www.contentstack.com/docs/developers/developer-hub/marketplace-app-boilerplate/

Reply:
keep all const in a separate file that way it would be easy in future to maintain and update